scripts/build-ipfs

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -e
+
+# Generating valid index.ipfs.html file
+sed 's/src\/index.ts/src\/index.ipfs.ts/g' ./index.html > ./index.ipfs.html
+
+vite build -c ./vite.config.ipfs.ts
+cp _redirects build/_redirects
+# TODO: Eventually rollup or vite can do the renaming part from index.ipfs.html to index.html
```

Comment:
This could be improved, if `vite build` would output the passed in `index.ipfs.html` directly as `index.html`